src/index.html

```diff
@@ -1,14 +1,99 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+  <!-- #region HEAD ------------------------------------>
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
13. Tag’s closing bracket should be on the same level as opening one.

Fix it in other places too.